testsuite/MDAnalysisTests/utils/test_modelling.py

```diff
@@ -55,12 +48,12 @@ def capping(ref, ace, nma, output):
     # TODO consider a case when the protein resid is 1 and all peptide has to be shifted by +1, put that in docs as a
     #  post-processing step
     alignto(ace, ref, select={
-            "mobile": "resid {0} and backbone".format(resid_min),
-            "reference": "resid {0} and backbone".format(resid_min)},
+        "mobile": "resid {0} and backbone".format(resid_min),
```

Comment:
This alignment is wrong. It should start on the opening brace. What alignment does yapf produce here?